docs/sources/administration/cli.md

```diff
@@ -10,41 +10,213 @@ weight = 8
-You can find more information about how to install and manage your plugins in the
-[plugins page]({{< relref "../plugins/installation.md" >}}).
+To list all commands and options:
+```
+grafana-cli -h
+```
+
+**Linux users**
+Some commands, such as installing or removing plugins, require `sudo` in order to run.
+
+**Windows users**
+Enter commands in Windows PowerShell, preceded by `.\`. They will not work on the regular command line.
+
+Some commands, such as installing or removing plugins, require you to run Windows PowerShell as Administrator. 
+
+Before you enter commands, `cd` into the plugins directory. The default path is:
+```
+cd "C:\Program Files\GrafanaLabs\grafana\bin"
+```
+
+## Grafana CLI command syntax
+
+The general syntax for commands in Grafana CLI is:
+```bash
+grafana-cli [global options] command [command options] [arguments...]
+```
+
+## Global options
 
-## Admin
+Grafana CLI allows you to temporarily override certain Grafana default settings. Except for `--help` and `--version`, most global options are only used by developers.
 
-> This feature is only available in Grafana 4.1 and above.
+Each global option applies only to the command in which it is used. For example, `--pluginsDir value` does not permanently change where Grafana saves plugins. It only changes it for command in which you apply the option.
 
-To show all admin commands:
-`grafana-cli admin`
+### Display Grafana CLI help
+
+`--help` or `-h` displays the help, including default paths and Docker configuration information.
+
+**Example:**
+```bash
+grafana-cli -h
+```
+
+### Display Grafana CLI version
+
+`--version` or `-v` prints the version of Grafana CLI currently running.
+
+**Example:**
+```bash
+grafana-cli -v
+```
+
+### Override default plugin directory
+
+`--pluginsDir value` overrides the path to where your local Grafana instance stores plugins. Use this option if you want to install, update, or remove a plugin somewhere other than the default directory ("/var/lib/grafana/plugins") [$GF_PLUGIN_DIR].
+
+**Example:**
+```bash
+grafana-cli --pluginsDir "/var/lib/grafana/devplugins" plugins install <plugin-id>
+```
+
+### Override default plugin repo URL
+
+`--repo value` allows you to download and install or update plugins from a repository other than the default Grafana repo.
+
+**Example:**
+```bash
+grafana-cli --repo "https://example.com/plugins" plugins install <plugin-id>
+```
+
+### Override default plugin .zip URL
+
+`--pluginUrl value` allows you to download a .zip file containing a plugin from a local URL instead of downloading it from the default Grafana source.
+
+**Example:**
+```bash
+grafana-cli --pluginUrl https://company.com/grafana/plugins/<plugin-id>-<plugin-version>.zip plugins install <plugin-id>
+```
+
+### Override Transport Layer Security
+
+`--insecure` allows you to turn off Transport Layer Security (TLS) verification (insecure). You might want to do this if you are downloading a plugin from a non-default source.
```

Comment:
We should note here that we don't recommend this as it is a security risk.

Reply:
Added a warning.